staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2390,6 +2400,11 @@ type ContainerStatus struct {
 	// Container's ID in the format 'docker://<container_id>'.
 	// +optional
 	ContainerID string `json:"containerID,omitempty" protobuf:"bytes,8,opt,name=containerID"`
+	// Specifies whether the container has passed its startup probe.
+	// Initialized as false, becomes true after startupProbe is considered successful.
+	// Resets to false when the container is restarted, or if kubelet loses state temporarily.
+	// Is always true when no startupProbe is defined.
+	Started *bool `json:"started" protobuf:"varint,9,opt,name=started"`
```

Comment:
this field must be `omitempty` and `// +optional`. that will remove the need to change roundtrip fixtures where new data was added to serialized responses from prior releases

Reply:
should be fine now, let's see if all tests pass